src/node/blockstorage.h

```diff
@@ -30,7 +30,6 @@
 #include <vector>
 
 class BlockValidationState;
-class CAutoFile;
```

Comment:
It's not buggy, because blockstorage.h includes dbwrapper.h which includes streams.h; but I think it should just be including streams.h directly, and that seems to be what iwyu is recommending?

Reply:
I don't think you're missing anything; but I don't really see the point of including a fwd declaration when that's not enough to be able to use the function, especially when the full header's already included indirectly? The tidy job says:

```
+#include "flatfile.h"
+#include "streams.h"
+#include "uint256.h"
```

which seems reasonable to me? When I run iwyu locally, it also seems to suggest primitives/block.h (rather than `class CBlock`), which is already included via chain.h anyway, so seems fine.

I've just pushed an update that gets closer to what iwyu suggests.